lombok.config

```diff
@@ -0,0 +1,2 @@
+config.stopBubbling = true
```

Comment:
This file prevents lombok annotations from being considered in coverage. It doesn't make sense to perform UTs for things we haven't written.